stacker/commands/stacker/info.py

```diff
@@ -0,0 +1,15 @@
+"""Gets information on the CloudFormation stacks based on the given config."""
+
+from .base import StackerCommand
+from ...actions import info
+
+
+class Info(StackerCommand):
+
+    name = 'info'
+    description = __doc__
+
+    def run(self, options, **kwargs):
+        super(Info, self).run(options, **kwargs)
+        action = info.Action(options.context, provider=options.provider)
```

Comment:
Probably worth adding an argument to get the output from a single stack if specified.